api/index.js

```diff
@@ -31,14 +32,23 @@ app.post("/compile", (req, res, next) => {
 
   var output = JSON.parse(solc.compile(JSON.stringify(input)));
   res.send(output);
-  // `output` here contains the JSON output as specified in the documentation
-  for (var contractName in output.contracts["test.sol"]) {
-    console.log(
-      contractName +
-        ": " +
-        output.contracts["test.sol"][contractName].evm.bytecode.object
-    );
-  }
+  console.log(output);
+});
+
+app.post("/deploy", (req, res, next) => {
```

Comment:
I don't think we need to deploy from the server. We want to use the user's wallet to deploy so we probably need to do it on the frontend where it's connected.